fe/src/main/java/org/apache/doris/common/proc/StatisticProcDir.java

```diff
@@ -145,6 +150,12 @@ public ProcResult fetchResult() throws AnalysisException {
                                     unhealthyTabletIds.put(dbId, tablet.getId());
                                 }
 
+                                // find tablet with no available replica;
+                                if (!tablet.hasAvailableReplica(infoService, partition.getVisibleVersion(),
```

Comment:
I think we can just add a new enum value in `TabletStatus`, for example, `NO_AVAILABLE_REPLICA`, and return it by calling `getHealthStatusWithPriority()` above. So that we can save some iteration.

But adding new enum value in `TabletStatus` may affect other code, which is uncontrollable. So I suggest to add a new parameter in `getHealthStatusWithPriority()`, eg, `getHealthStatusWithPriority(..., bool returnNoAvlExplicit)`.
And if `returnNoAvlExplicit` is true, it return return `NO_AVAILABLE_REPLICA` explicit.